### PR TITLE
Fix bug where pickFields does not work correctly with dot notation

### LINKED ISSUE #58: pickFields does not work correctly with dot notation
Seems `lodash.pick` is unmaintained and has bugs compared to the current version.

```javascript
const doc2 = {
  "source": {
    "name": "foo",
    "event": "bar",
    "data": {
      "businessUnit": {
        "name": "baz"
      }
    }
  }
};

const pick1 = require('lodash.pick');
const pick2 = require('lodash/pick');

console.log('lodash.pick:', pick1(doc2, [
  'source.name',
  'source.event',
  'source.data.businessUnit.name'
]));

console.log('lodash/pick:', pick2(doc2, [
  'source.name',
  'source.event',
  'source.data.businessUnit.name'
]));
```

Result:

```
lodash.pick: {}
lodash/pick: { source: { name: 'foo', event: 'bar', data: { businessUnit: [Object] } } }
```

### PR DESCRIPTION
**Pull request by compwright**
* Adds test case
* Updates dependencies

Resolves #58

### CODE REVIEW

File: lib/handler.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const bunyan = require('alpha-lambda-bunyan');
 const DynamoDB = require('aws-sdk/clients/dynamodb');
 const lambdaHandler = require('alpha-lambda');
-const pick = require('lodash.pick');
+const pick = require('lodash/pick');
```
Comment (AntonBazhal) [inline lib/handler.js:4]: How is this different from bringing the whole `lodash` library?

Comment (compwright) [reply lib/handler.js:4]: It allows for tree-shaking to reduce the overall package size when bundling with webpack or a similar tool. Including the whole library like this:

```javascript
const { pick } = require('lodash');
```

causes the entire lodash library to be included in the bundle, whether it's actually used in application code or not.

Comment (AntonBazhal) [reply lib/handler.js:4]: I get that, but it's still not ideal. I think there should be a defect raised against `lodash`.

**Review by AntonBazhal — APPROVED** [review]
I'm OK to have `lodash` until they fix `lodash.pick`

**Pull request merged**